docusaurus.config.js

```diff
@@ -63,6 +63,43 @@ module.exports = {
         },
       },
     ],
+    [
+      "@docusaurus/plugin-content-docs",
+      {
+        id: "api-operate-docs",
+        path: "api/operate",
+        routeBasePath: "api/operate",
+        sidebarPath: require.resolve("./api/operate/operate_sidebars.js"),
+        editUrl: "https://github.com/camunda/camunda-docs/edit/main/",
+        lastVersion: "current",
+        versions: {
+          current: {
+            label: "1.0",
+            path: "",
+          },
+        },
+        docLayoutComponent: "@theme/DocPage",
+        docItemComponent: "@theme/ApiItem",
+      },
+    ],
```

Comment:
This is a new docs instance, designed to hold only the Operate OpenAPI experience. I would anticipate that each API we give an interactive experience will get its own docs instance, because the APIs may version independently.

Reply:
This is related to this epic around versioning our public APIs - https://github.com/camunda/product-hub/issues/1876